x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/analysis/Analyzer.java

```diff
@@ -99,7 +108,9 @@
 import static org.elasticsearch.xpack.ql.type.DataTypes.KEYWORD;
 import static org.elasticsearch.xpack.ql.type.DataTypes.LONG;
 import static org.elasticsearch.xpack.ql.type.DataTypes.NESTED;
+import static org.elasticsearch.xpack.ql.type.DataTypes.NULL;
```

Comment:
Could we not import `DataTypes.NULL`? It just reads too much like regular `null` to me and I get confused. I think it'd be double confusing for C/C++ people because they write regular `null` in all caps.